tools/mksurfdata_map/src/mkdomainMod.F90

```diff
@@ -866,15 +866,6 @@ subroutine domain_checksame( srcdomain, dstdomain, tgridmap )
      end if
      do n = 1,ns
         ni = src_indx(n)
-        if ( srcdomain%maskset )then
-           if (srcdomain%mask(ni) /= mask_src(ni)) then
-              write(6,*) trim(subname)// &              
-                 ' ERROR: input domain mask and gridmap mask are not the same at ni = ',ni
-              write(6,*)' domain  mask= ',srcdomain%mask(ni)
-              write(6,*)' gridmap mask= ',mask_src(ni)
-              call abort()
-           end if
-        end if
```

Comment:
Removed this error check confirming that the input domain mask should equal the gridmap mask because the gridmap mask (SRC mask found in the "nomask" mapping file) now equals 1 everywhere.

Reply:
Yep.